hpx/runtime.hpp

```diff
@@ -345,6 +345,12 @@ namespace hpx
         boost::scoped_ptr<components::server::memory> memory_;
         boost::scoped_ptr<components::server::runtime_support> runtime_support_;
     };
+    ///////////////////////////////////////////////////////////////////////////
+    /// Keep the factory object alive which is responsible for the given
+    /// component type. This a purely internal function allowing to work
+    /// around certain library specific problems related to dynamic
+    /// loading of external libraries.
+    HPX_EXPORT bool tolerate_node_faults();
```

Comment:
This comment does not match the code.

Reply:
sorry, copy/paste :-( I'll add a proper comment

Reply:
@KADichev ping?

Reply:
I just updated the branch. Not sure if anything else is required of me here. Never did PR before.